test_helpers/fixtures/main_test_file.org

```diff
@@ -12,9 +12,12 @@ Some description content
 
 * A header with tags                                              :tag1:tag2:
 * A header with [[https://organice.200ok.ch][a link]]
-* A header with a link to a local .org file as content
-
-  [[file:schedule_and_timestamps.org][a local .org file]]
+* A header with various links as content
+  Relative link to [[file:schedule_and_timestamps.org][an existing .org file in the same directory]]
+  Relative link to [[file:subdir/foo.org][a fictitious .org file in a sub-directory]]
```

Comment:
For completeness, add relative `../` link?

Reply:
That was supposed to be on the line immediate below this comment, but Orgmode auto-corrected it to my local absolute path without me noticing :-(  Will fix.

Reply:
That's interesting behavior. I have had the same in my tests!